webgl/lessons/ko/webgl-resizing-the-canvas.md

```diff
@@ -84,8 +84,8 @@ canvas 크기 설정하는데 canvas 크기에 영향을 주는 CSS가 없다면
       var displayHeight = canvas.clientHeight;
 
       // canvas가 같은 크기가 아닐 때 확인
-      if (canvas.width  != displayWidth ||
-          canvas.height != displayHeight) {
+      if (canvas.width  != displayWidth
```

Comment:
If I'm being picky then this was the prefered way before, in particular operators go at the end of a line (https://google.github.io/styleguide/jsguide.html#formatting-line-wrapping)  although i'll be you can find a case somewhere in the code where I put them in the front because I thought it aligned better.

I'm more of alignment oriented person when things are related. As such you can see the extra space after `width` is meant the align the 2 lines

```
  if (canvas.width  != displayWidth ||
      canvas.height != displayHeight) {
```

I personally find that easier to read though it's discouraged in Google's style